src/__tests__/applicants.int.test.ts

```diff
@@ -1,40 +1,32 @@
-import request from 'supertest';
-import { TinyPgError } from 'tinypg';
-import { app } from '../app';
-import {
-  db,
-  loadTableMetrics,
-} from '../database';
-import { getLogger } from '../logger';
-import { PostgresErrorCode } from '../types';
-import { expectTimestamp } from '../test/utils';
-import { mockJwt as authHeader } from '../test/mockJwt';
-import type { Result } from 'tinypg';
+import request from "supertest";
```

Comment:
It looks like we're switching from single quotes (`'`) to double quotes (`"`) throughout. Is that an intentional change? Is that configurable?

Reply:
It's what prettier defaults to. My approach to prettier in general (and this PR) is to not think about their choices and just accept them.

Reply:
I was also promised that everyone would begin to use proper typographic apostrophes from now on. I couldn’t be happier.